implementation/alpha/node/node.ino

```diff
@@ -113,7 +120,10 @@ void setup() {
     rf69_manager.setRetries(0);
     rf69_manager.setTimeout(10);
 
-    last_reading_time = millis();
+  // generate on startup
+  if (do_first_health_packet) health_packet_generate();
+  if (do_first_reading_packet) data_packet_generate();
```

Comment:
This should be if(my_data.has_sensor && do_first_reading_packet).
This is duplicated of line 334. line 334 should remove do_first_reading_packet.